src/main/java/com/hivemq/extensions/rbac/FileAuthAuthenticator.java

```diff
@@ -32,15 +34,27 @@
 
 public class FileAuthAuthenticator implements SimpleAuthenticator {
 
-    private final @NotNull
-    CredentialsValidator credentialsValidator;
+    private final @NotNull CredentialsValidator credentialsValidator;
 
-    FileAuthAuthenticator(final @NotNull CredentialsValidator credentialsValidator) {
+    private final @NotNull ExtensionConfig extensionConfig;
+
+    FileAuthAuthenticator(final @NotNull CredentialsValidator credentialsValidator, final @NotNull ExtensionConfig extensionConfig) {
         this.credentialsValidator = credentialsValidator;
+        this.extensionConfig = extensionConfig;
     }
 
     @Override
     public void onConnect(@NotNull final SimpleAuthInput simpleAuthInput, @NotNull final SimpleAuthOutput simpleAuthOutput) {
+        final List<String> listenerNames = extensionConfig.getListenerNames();
```

Comment:
It would be better if this would be a Set instead of List, because the contains check is cheaper and you would automatically eliminate duplicates